server/src/main/java/org/apache/iotdb/db/engine/tsfilemanagement/utils/HotCompactionUtils.java

```diff
@@ -53,32 +55,23 @@
 public class HotCompactionUtils {
 
   private static final Logger logger = LoggerFactory.getLogger(HotCompactionUtils.class);
+  private static final int mergePagePointNum = IoTDBDescriptor.getInstance().getConfig()
+      .getMergePagePointNumberThreshold();
 
   private HotCompactionUtils() {
     throw new IllegalStateException("Utility class");
   }
 
-  private static Pair<ChunkMetadata, Chunk> writeSeqChunk(String storageGroup,
-      Map<String, TsFileSequenceReader> tsFileSequenceReaderMap, String deviceId,
-      String measurementId,
-      List<TsFileResource> levelResources)
-      throws IOException {
+  private static Pair<ChunkMetadata, Chunk> readByAppendMerge(RateLimiter compactionReadRateLimiter,
```

Comment:
I think some comments should be added, for the software is open sourced

Reply:
okay, I will add some comment